connectors/sources/mongo.py

```diff
@@ -17,32 +17,50 @@ class MongoDataSource(BaseDataSource):
 
     def __init__(self, configuration):
         super().__init__(configuration=configuration)
-        self.client = AsyncIOMotorClient(
-            self.configuration["host"],
-            directConnection=True,
-            connectTimeoutMS=120,
-            socketTimeoutMS=120,
-        )
+
+        client_params = {}
+
+        host = self.configuration["host"]
+        user = self.configuration["user"]
+        password = self.configuration["password"]
+
+        if self.configuration["direct_connection"]:
+            client_params["directConnection"] = True
+
+        if len(user) > 0 or len(password) > 0:
+            client_params["username"] = user
+            client_params["password"] = password
+
+        print(f"host: {host}, params: {client_params}")
```

Comment:
Should this be `logger.info(f"host: {host}, params: {client_params}")`?

Reply:
That's a leftover from debugging, will delete it, good catch!